docs: update migration guide (#1751)
kyma-project/api-gateway: docs/user/custom-resources/apirule/04-60-apirule-migration.md

## Migration Procedure

The migration procedure consists of the following steps, which are executed in a time-separated manner, with a one-minute delay between each step:

> 1. The resource owner updates the APIRule to version `v2`. As an immediate result, new Istio Authorization Policy and Istio Authentication Policy resources are created.

Reviewer: ```suggestion
1.  As the resource owner, you must update the APIRule to version `v2`. As an immediate result, new Istio Authorization Policy and Istio Authentication Policy resources are created.
```

> 2. To retain the APIRule `v1` CORS configuration, update the APIRule with the CORS configuration.
> 3. To retain APIRule `v1` internal traffic policy, apply the following AuthorizationPolicy. Remember to change the selector label to the one pointing to the target workload:

Reviewer: ```suggestion
2. To retain the APIRule `v1beta1` CORS configuration, update the APIRule with the CORS configuration.
3. To retain APIRule `v1beta1` internal traffic policy, apply the following AuthorizationPolicy. Remember to change the selector label to the one pointing to the target workload:
```

The migration procedure consists of the following steps, which are executed in a time-separated manner, with a one-minute delay between each step:

> 1. The resource owner updates the APIRule to version `v2alpha1`. As an immediate result, new Istio Authorization Policy and Istio Authentication Policy resources are created.

Reviewer: ```suggestion
1. As the resource owner, you must update the APIRule to version `v2alpha1`. As an immediate result, new Istio Authorization Policy and Istio Authentication Policy resources are created.
```

> 2. To retain APIRule V1 CORS configuration, the resource owner needs to update the APIRule with the CORS configuration.

Reviewer: ```suggestion
2. To retain the APIRule `v1` CORS configuration, update the APIRule with the CORS configuration.
```

> 3. To retain APIRule V1 internal traffic policy, Apply the following AuthorizationPolicy. Remember to change selector label to the one pointing to the target workload:

Reviewer: ```suggestion
3. To retain APIRule `v1` internal traffic policy, apply the following AuthorizationPolicy. Remember to change the selector label to the one pointing to the target workload:
```

    ```yaml
    apiVersion: security.istio.io/v1
    kind: AuthorizationPolicy
    metadata:
      name: allow-internal
      namespace: ${NAMESPACE}
    spec:
      selector:
        matchLabels:
          ${KEY}: ${TARGET_WORKLOAD}
      action: ALLOW
      rules:
      - from:
        - source:
            notPrincipals: ["cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"]
    ```

> 4. The Istio VirtualService resource is updated to point directly to target Service, bypassing Ory Oathkeeper.

Reviewer: ```suggestion
4. The Istio VirtualService resource is updated to point directly to the target Service, bypassing Ory Oathkeeper.
```
